tests/vmi_configuration_test.go

```diff
@@ -1732,6 +1735,23 @@ var _ = Describe("[sig-compute]Configurations", decorators.SigCompute, func() {
 				Expect(origVmiWithoutHeadroom.Spec.Domain.Memory).To(Equal(vmiWithoutHeadroom.Spec.Domain.Memory), "vmi guest memory is not expected to change")
 			})
 		})
+
+		It("be able to use hyperv passthrough and ensure it's non-migratable", func() {
```

Comment:
Should this be moved inside hyperv_test file?

Reply:
Yeah I was conflicting on it as well, but I think you're right.

I've made a small refactoring for the hyperv file. Along the way I've removed the `Serial` label from more than half of the tests. So your feedback actually helped improve CI performance in general :) thank you!

Reply:
:slightly_smiling_face: